lib/core/src/lib/notifications/components/notification-history.component.scss

```diff
@@ -28,6 +28,8 @@
         &-message,
         &-no-message {
             font-size: var(--theme-body-1-font-size) !important;
+            white-space: normal !important;
```

Comment:
Can we avoid these !important somehow?

Reply:
Tried

Reply:
hmm I tried to set in browser that here: 
<img width="1062" alt="image" src="https://user-images.githubusercontent.com/115619721/216310796-82cfa6b2-bc2c-44cb-a674-713b3c1dba1f.png">
and then text takes two lines instead of being cut.